deploy/helm/sumologic/README.md

```diff
@@ -20,6 +20,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
+| `sumologic.metrics.provider`                                              | Set provider to use for metrics forwarding and metadata enrichment.                                                                                                                                                                                                                                                                                                                                                                           | `fluentd`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
I think that it is worth to mention here that provider can be set either to "fluentd" or "otelcol".